bin/.cicd-cli.py

```diff
@@ -13,16 +14,57 @@
 
 sys.path.insert(1, ci_dir)
 
-import clean
-import glci.util
-import glci.model
-import paths
+import clean      # noqa: E402
+import glci.util  # noqa: E402
+import glci.model # noqa: E402
+import paths      # noqa: E402
+
+
+# see also:
+# https://stackoverflow.com/questions/43968006/support-for-enum-arguments-in-argparse/55500795
+class EnumAction(argparse.Action):
+    """
+    Argparse action for handling Enums
+    """
+    def __init__(self, **kwargs):
+        # Pop off the type value
+        enum_type = kwargs.pop("type", None)
+
+        # Ensure an Enum subclass is provided
+        if enum_type is None:
+            raise ValueError("type must be assigned an Enum when using EnumAction")
+        if not issubclass(enum_type, enum.Enum):
+            raise TypeError("type must be an Enum when using EnumAction")
+
+        # Generate choices from the Enum
+        kwargs.setdefault("choices", tuple(e.value for e in enum_type))
+
+        super(EnumAction, self).__init__(**kwargs)
+
+        self._enum = enum_type
+
+    def __call__(self, parser, namespace, values, option_string=None):
+        # Convert value back into an Enum
+        value = self._enum(values)
+        setattr(namespace, self.dest, value)
 
 
 def clean_build_result_repository():
-    parser = argparse.ArgumentParser()
-    parser.add_argument('--cicd-cfg', default='default')
-    parser.add_argument('--snapshot-max-age-days', default=30, type=int)
+    parser = argparse.ArgumentParser(
+        description='Cleanup in manifests repository (S3)',
+        epilog='Warning: dangerous, use only if you know what you are doing!',
+    )
+    parser.add_argument(
+        '--cicd-cfg',
+        default='default',
+        help='configuration key for ci, default: \'%(default)s\'',
```

Comment:
Displaying the default can also be done with argparse by passing `ArgumentDefaultsHelpFormatter` as formatter to `ArgumentParser`